backend/metamodel/syson-sysml-metamodel/src/main/java/org/eclipse/syson/sysml/impl/ReferenceUsageImpl.java

```diff
@@ -69,5 +72,18 @@ public EList<TextualRepresentation> getTextualRepresentation() {
         List<TextualRepresentation> textualRepresentation = new ArrayList<>();
         return new EcoreEList.UnmodifiableEList<>(this, SysmlPackage.eINSTANCE.getElement_TextualRepresentation(), textualRepresentation.size(), textualRepresentation.toArray());
     }
+    
+    @Override
+    public String getDeclaredName() {
+        Element foundActionUsage = this.getOwnedFeatureMembership().stream()
```

Comment:
I'm not sure about that one. The ReferenceUsage is widely used in sysmlv2, for a large set of usages, not only ActionUsages.
You code should work for other kind of Usages too.